templates/repo/migrate/migrate.tmpl

```diff
@@ -3,9 +3,9 @@
 	<div class="ui middle very relaxed page grid">
 		<div class="column">
 			{{template "repo/migrate/helper" .}}
-			<div class="ui three stackable cards">
+			<div class="ui cards migrate-entries">
```

Comment:
It seems that most styles are provided by `migrate-entries` and `migrate-entry` now? Is "ui cards" still necessary?

Reply:
`.ui.cards` is necessary for styling aspecs like border and background and more.